dashboard/app/models/user.rb

```diff
@@ -272,12 +295,56 @@ class User < ApplicationRecord
 
   has_many :authentication_options, dependent: :destroy
   accepts_nested_attributes_for :authentication_options
```

Comment:
I assume these were coupled as well. Where would the `accepts_nested_attributes_for` would go?

Reply:
Yeah, I see these two as coupled. `accepts_nested_attributes_for` specifies that `authentication_options` can be "nested" when creating a User. [Here's where](https://github.com/code-dot-org/code-dot-org/blob/63d5581a4a62fde72ea495660683479d0a93eb9f/dashboard/lib/services/lti.rb#L22-L29) we're doing that.